src/Sylius/Component/Core/Customer/Statistics/CustomerOrdersStatistics.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Customer\Statistics;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class CustomerOrdersStatistics
```

Comment:
I'd simply say "PerChannelCustomerStatistics", we may add some stats not related to the orders.